security-analysis/security-analysis-api/src/main/java/com/powsybl/security/limitreduction/computation/SimpleLimitsReducer.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.security.limitreduction.computation;
+
+import com.powsybl.iidm.network.LoadingLimits;
+
+/**
+ *
```

Comment:
Add documentation here:
Limits reducer applying the same reduction value for the permanent and every temporary limits.